python/tidy/servo_tidy/tidy.py

```diff
@@ -19,6 +19,25 @@
 import sys
 from licenseck import MPL, APACHE, COPYRIGHT, licenses_toml, licenses_dep_toml
 import colorama
+import toml
+
+# Load configs from servo-tidy.toml if config file exists
+if os.path.exists("./servo-tidy.toml"):
```

Comment:
Also, why have you split the config-file-reading logic between the top level here and the `set_configs` function down at L86?